lib/presentation/widgets/background_widget.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:askm/generated/assets/assets.gen.dart';
+import 'package:flutter/material.dart';
+
+class BackgroundWidget extends StatelessWidget {
+  const BackgroundWidget({
+    super.key,
+    required this.child,
+  });
+
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: [
+        Image.asset(
+          Assets.images.backgroundImage.keyName,
+          fit: BoxFit.cover,
+          width: double.infinity,
```

Comment:
are you shoure whay you need to write wigth and height?

Reply:
Yes i am